### DISCUSSION TITLE
Adding events from a mobile phone

### DISCUSSION THREAD

**Discussion by Manners13** [craigk5n/webcalendar#481] [General]
HI,

We have used webcal for a long time now albeit a very old version. we have however installed the new version to try to migrate over to it  but slowly trying to work through a few problems. In old versions the event boxes used to have a little + which you could tap on a mobile phone to be able to add an entry, The new version has removed the + and enabled double click to add event... on a mobile phone this doesnt work if you tap it does nothing then if you double tap it zooms in on the page. Is there a way to re add the + back into the boxes?

**Comment by Combi44** [comment]
I have the same problem. Have you found a solution?
